### PR TITLE
fix : remove memo if no need

### PR DESCRIPTION
**Pull request by ZilongRRR**
# Description

* Remove memo
  * fee in cat send
  * change in xch send

image

Fixes # (issue)

* mutli memo while cat send with fee

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/store/TransactionStore.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ class TransactionStore {
             const XCHspendsList = await Wallet.generateXCHSpendList({
                 puzzleReveal,
                 amount: '0',
-                memo,
+                memo: '', // fee no need memo
```
Comment (hryjosn) [inline src/store/TransactionStore.ts]: no need => memo is unnecessary for fee

File: src/utils/Wallet.ts [hunk 2]
```diff
@@ -315,10 +315,10 @@ export class Wallet extends Program {
             })
         }
 
+        // change no need memo
```
Comment (hryjosn) [inline src/utils/Wallet.ts]: no need => memo is unnecessary for fee

**Review by hryjosn — CHANGES_REQUESTED** [review]

File: src/store/TransactionStore.ts (hunk 1 above)
Comment (ZilongRRR) [reply src/store/TransactionStore.ts]: done
[45a030a](https://github.com/hashgreen/hoogii-wallet/pull/19/commits/45a030a02e34176c57a35cae449ddc3bd94a74d0)

File: src/utils/Wallet.ts (hunk 2 above)
Comment (ZilongRRR) [reply src/utils/Wallet.ts]: done 
[45a030a](https://github.com/hashgreen/hoogii-wallet/pull/19/commits/45a030a02e34176c57a35cae449ddc3bd94a74d0)

**Review by hryjosn — APPROVED** [review]

**Pull request merged**